demos/mqtt/mqtt_demo_plaintext/config.h

```diff
@@ -22,8 +22,9 @@
 #ifndef CONFIG_H
 #define CONFIG_H
 
-/* Set network context to socket (int). */
-typedef int MQTTNetworkContext_t;
+/* Set network context to OpenSSL SSL context. */
+#include <openssl/ssl.h>
```

Comment:
Why where these changes to the config file needed in this PR?

Reply:
My mistake. I copied the wrong config.h file onto here. Thanks.